### PR TITLE
Minor Tweaks

### PR DESCRIPTION
**Pull request by jshaughn**
- remove some leftover cassandra references
- fix small issue in severities query

@lucasponce A few small cleanups I came across when working on 1.8.

### CODE REVIEW

File: engine/pom.xml [hunk 1]
```diff
@@ -48,13 +48,13 @@
 
     <dependency>
       <groupId>org.hawkular.alerts</groupId>
-      <artifactId>hawkular-alerts-filter-api</artifactId>
+      <artifactId>hawkular-alerts-events-aggregation</artifactId>
```
Comment (lucasponce) [inline engine/pom.xml]: mmm ?

File: engine/pom.xml [hunk 2]
```diff
@@ -48,13 +48,13 @@
...
       <version>${project.version}</version>
     </dependency>
 
     <dependency>
       <groupId>org.hawkular.alerts</groupId>
-      <artifactId>hawkular-alerts-events-aggregation</artifactId>
+      <artifactId>hawkular-alerts-filter-api</artifactId>
```
Comment (lucasponce) [inline engine/pom.xml]: Same mmm ?

**Comment by lucasponce** [comment]
Good clean-up of old references.
Merging.

File: engine/pom.xml (hunk 2 above)
Comment (jshaughn) [reply engine/pom.xml]: yes, no changes to the deps, just flipped the order to be alphabetical.  Probably didn't need to do that :)

**Pull request merged**